src/Raven.Server/Smuggler/Documents/DatabaseDestination.cs

```diff
@@ -1417,9 +1417,13 @@ private void AddTrxnIfNeeded(DocumentsOperationContext context, string id, ref s
                         }
                     }
 
-                    var trxn = ChangeVectorUtils.GetEtagById(oldChangeVector, _database.ClusterTransactionId);
-                    if (trxn > 0)
-                        changeVector += $",TRXN:{trxn}-{_database.ClusterTransactionId}";
+                    //The ClusterTransactionId can be null if the database was migrated from version smaller then v5.2 
+                    if (_database.ClusterTransactionId != null)
```

Comment:
This is happening when importing to a database that was created before the upgrade?

Reply:
Yes